proto/cosmos/gov/v1beta1/gov.proto

```diff
@@ -46,7 +46,7 @@ message TextProposal {
 message Deposit {
   option (gogoproto.equal) = true;
 
-  uint64   proposal_id = 1 [(gogoproto.customname) = "ProposalID", (gogoproto.moretags) = "yaml:\"proposal_id\""];
+  uint64   proposal_id = 1 [(gogoproto.moretags) = "yaml:\"proposal_id\""];
```

Comment:
I think we need a change log for non-ibc changes since it is api-breaking?

Reply:
yes we should add one